packages/calypso-e2e/src/lib/pages/people-page.ts

```diff
@@ -34,11 +41,62 @@ export class PeoplePage {
 	 * @returns {Promise<void>} No return value.
 	 */
 	async clickTab( name: PeoplePageTabs ): Promise< void > {
-		// These navtabs do not change into a dropdown, unlike the navtabs present
-		// elsewhere in Calypso (eg. MediaPage).
-		await this.page.click( `${ selectors.navTabs } span:text("${ name }")` );
+		const navTabs = await this.page.waitForSelector( selectors.navTabs );
+
+		const isDropdown = await navTabs
+			.getAttribute( 'class' )
+			.then( ( value ) => value?.includes( 'is-dropdown' ) );
+
+		if ( isDropdown ) {
+			// Mobile view - navtabs become a dropdown.
+			await navTabs.click();
+			await this.page.click( `${ selectors.navTabsDropdownOptions } >> text=${ name }` );
+		} else {
+			// Desktop view - navtabs are constantly visible tabs.
+			await this.page.click( `${ selectors.navTabs } >> text=${ name }` );
+		}
+	}
+
+	/* Team People */
+
+	/**
+	 * Locate and click on an user.
+	 *
+	 * @param {string} username Username of the user.
+	 */
+	async selectUser( username: string ): Promise< void > {
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click( selectors.teamUser( username ) ),
+		] );
+	}
+
+	/**
+	 * Delete the user from site.
+	 */
+	async deleteUser(): Promise< void > {
+		const elementHandle = await this.page.waitForSelector(
+			selectors.deletedUserContentAction( 'delete' )
+		);
+
+		// Invoke scroll directly via JavaScript.
+		// Playwright's built-in `click` or `check` methods are not able to
+		// scroll the radio buttons into view for uncertain reasons.
+		await this.page.evaluate(
+			( element: SVGElement | HTMLElement ) => element.scrollIntoView(),
+			elementHandle
```

Comment:
As noted in the comments, strangely Playwright isn't able to scroll this page - even with `elementHandle.scrollIntoViewIfNeeded`.